cinema/views.py

```diff
@@ -127,6 +132,38 @@ def upload_image(self, request, pk=None):
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
In the 'upload_image' method, it's good to see that you're validating the serializer before saving. However, it would be better to provide a more specific error message when the serializer is not valid. This will help the client understand what went wrong.